test/ts-benchmarks/selectFrom.bench.ts

```diff
@@ -0,0 +1,95 @@
+import { bench } from '@ark/attest'
+import type { DB } from '../typings/test-d/huge-db.test-d'
+import type { Kysely } from '../../dist/esm/index.js'
+
+declare const kysely: Kysely<DB>
+declare const knex: Kysely<any>
```

Comment:
Calling this `knex` is a bit confusing. Maybe just `kyselyAny` or something?

Reply:
You're right. Done.